packages/app-store/cal-ai/config.json

```diff
@@ -14,5 +13,10 @@
   "isTemplate": false,
   "__createdUsingCli": true,
   "__template": "basic",
-  "dirName": "cal-ai"
+  "dirName": "cal-ai",
+  "paid": {
+    "price": 25,
+    "priceId": "price_1O1ziDH8UDiwIftkDHp3MCTP",
+    "mode": "subscription"
```

Comment:
It is not clear to me here whether it's per month or per year or something else.